retem and kvaris side stories

allister-translator opened the pull request:
early PR so dabir can get a head-start on proofreading

allister-translator committed:
Update and rename Files/ssc_501060.csv to Translated/Story/Side Aina/ssc_501060.csv

allister-translator committed:
Update ssc_501060.csv

allister-translator committed:
Update and rename Files/ssc_502060.csv to Translated/Story/Side Manon/ssc_502060.csv

DabirSA committed:
Update ssc_501060.csv

DabirSA committed:
Update ssc_502060.csv

DabirSA commented on Translated/Story/Side Manon/ssc_502060.csv:
```diff
@@ -0,0 +1,36 @@
+ssc_502060_0240#0,"""Have you tried asking Nadereh?"""
+ssc_502060_0250#0,"""... No, sorry, ignore that.<br>I got caught up in speculation."""
+ssc_502060_0260#0,"""I'll have to investigate just a bit more."""
+ssc_502060_0270#0,"""That's... true, but I doubt even she herself<br>would have a theory as to how it works."""
```
This line follows on weirdly from what I think is the player dialogue it's responding to. "Have you tried asking Nadereh?" "That's... true".

allister-translator replied:
fixt

allister-translator committed:
Update and rename Files/ssc_506010.csv to Translated/Story/Side Nadereh/ssc_506010.csv

allister-translator committed:
Update ssc_502060.csv

allister-translator committed:
Update and rename Files/ssc_501070.csv to Translated/Story/Side Aina/ssc_501070.csv

allister-translator committed:
Update ssc_501070.csv

allister-translator committed:
boomer hadi (info in description)

for context, he's telling a story that happened in the past, but he's talking about it in present-tense. you know like those people who go "So I'm standing before this guy, and I say to him," etc etc tl;dr it was a fucking headache to translate, so just do whatever if it sounds awkward anywhere

DabirSA committed:
Update ssc_501070.csv

DabirSA committed:
Update ssc_507010.csv

DabirSA committed:
Update ssc_506010.csv

allister-translator committed:
Update and rename Files/ssc_507030.csv to Translated/Story/Side Hadi/ssc_507030.csv

allister-translator committed:
Update and rename Files/ssc_503060.csv to Translated/Story/Side Bruder/ssc_503060.csv

allister-translator committed:
Update ssc_503060.csv

allister-translator committed:
Update and rename Files/ssc_507020.csv to Translated/Story/Side Hadi/ssc_507020.csv

allister-translator committed:
Update and rename Files/ssc_506020.csv to Translated/Story/Side Nadereh/ssc_506020.csv

allister-translator committed:
Update and rename Files/ssc_506030.csv to Translated/Story/Side Nadereh/ssc_506030.csv

allister-translator committed:
Update ssc_506030.csv

allister-translator committed:
Update ssc_506030.csv

allister-translator committed:
Update ssc_506030.csv

allister-translator committed:
Update and rename Files/ssc_504040.csv to Translated/Story/Side Crawford/ssc_504040.csv

DabirSA committed:
Update ssc_503060.csv

DabirSA committed:
Update ssc_504040.csv

allister-translator committed:
Update and rename Files/ssc_505020.csv to Translated/Story/Side Ran/ssc_505020.csv

allister-translator committed:
Update ssc_505020.csv

allister-translator committed:
Update and rename Files/ssc_505030.csv to Translated/Story/Side Ran/ssc_505030.csv

allister-translator committed:
Update ssc_505030.csv

allister-translator committed:
Update and rename Files/ssc_501080.csv to Translated/Story/Side Aina/ssc_501080.csv

allister-translator committed:
Update ssc_501080.csv

allister-translator committed:
Update and rename Files/ssc_501090.csv to Translated/Story/Side Aina/sc_501090.csv

allister-translator committed:
Update sc_501090.csv

allister-translator committed:
Update and rename Files/ssc_502070.csv to Translated/Story/Side Manon/ssc_502070.csv

allister-translator committed:
Update ssc_502070.csv

allister-translator committed:
Update and rename Files/ssc_509020.csv to Translated/Story/Side Meri/ssc_509020.csv

allister-translator committed:
Update and rename Files/ssc_509030.csv to Translated/Story/Side Meri/ssc_509030.csv

allister-translator committed:
Update ssc_509030.csv

allister-translator committed:
Update ssc_509030.csv

DabirSA committed:
Update ssc_501080.csv

DabirSA committed:
Update ssc_507030.csv

DabirSA committed:
Update ssc_509020.csv

allister-translator committed:
Update and rename Files/ssc_508010.csv to Translated/Story/Side Ilma/ssc_508010.csv

DabirSA committed:
Update ssc_507020.csv

DabirSA committed:
Update sc_501090.csv

DabirSA committed:
Update ssc_508010.csv

allister-translator committed:
Create ssc_508010.csv

DabirSA committed:
Update ssc_508010.csv

allister-translator committed:
Update ssc_508010.csv

DabirSA committed:
Update ssc_502070.csv

DabirSA committed:
Update ssc_506030.csv

DabirSA committed:
Update ssc_506020.csv

DabirSA committed:
Update ssc_505020.csv

DabirSA committed:
Update ssc_505030.csv

allister-translator committed:
Update ssc_508010.csv

allister-translator committed:
Update ssc_508010.csv

allister-translator committed:
Update ssc_508010.csv

allister-translator committed:
Update and rename Files/ssc_508020.csv to Translated/Story/Side Ilma/ssc_508020.csv

allister-translator committed:
Update and rename Files/ssc_502080.csv to Translated/Story/Side Manon/ssc_502080.csv

allister-translator committed:
Update ssc_502080.csv

allister-translator committed:
Rename sc_501090.csv to ssc_501090.csv

DabirSA committed:
Update ssc_508010.csv

DabirSA committed:
Update ssc_508020.csv

DabirSA committed:
Update ssc_502080.csv
